src/API/Helpers/InformationHeaders.php

```diff
@@ -2,19 +2,29 @@
 
 namespace Auth0\SDK\API\Helpers;
 
+/**
+ * Class InformationHeaders
+ * Builds, extends, modifies, and formats SDK telemetry data.
+ *
+ * @package Auth0\SDK\API\Helpers
+ */
 class InformationHeaders
 {
 
     /**
+     * Default header data to send.
      *
      * @var array
      */
-    protected $data = [];
+    protected $data = [ 'name' => 'Not set (PHP)', 'version' => 'Not set (PHP)' ];
```

Comment:
Default package data to call out if `setPackage` is not being called. Worth it to look into setting these in a constructor at some point in the future.

Reply:
@lbalmaceda - Can't do a transformation without a name present. I think this will help to figure out if we have untracked entries and where they came from.

Reply:
On second thought ... I think you're right. This is jumping the gun on addressing untracked SDKs. I'll pull this out tomorrow, just me know if you have any other feedback.

Reply:
In PHP, why wouldn't you be able to ask for the PHP version? 
http://php.net/manual/en/function.phpversion.php

Reply:
@cocojoe - This is the SDK version and has to be set explicitly.